### ISSUE TITLE
Can the 'last seen date and registered status' entries not have a border-top property?

### ISSUE THREAD

**Issue by darkred** [soscripted/sox#117]
### Installed Version: 2.0.1  Environment: Firefox/Greasemonkey

---
### Suggestion

The 'last seen date and registered status' entries not to have a `border-top` property
in order to be easier distinguishable that they are info for the question/answer author above
and that they not another comment.

i.e. add something like this CSS rule:
```
.comments {
	border-top: none !important;
}
```

Also, to further distinguish these entries from comments I'd suggest adding something like this rule:
```
.comments > table:nth-child(1) > tbody:nth-child(1) > tr:nth-child(1) > td:nth-child(2) > div:nth-child(1) {
	opacity: 0.4 !important;	
}
```

**Comment by shu8** [comment]
@darkred fixed in [v2.0.2 dev](https://github.com/soscripted/sox/raw/dev/sox.user.js) :)

I'm not completely sure with the opacity, but I've added it and I'll give it a chance this version and I'll come back to it in v2.0.3. this is what it looks like:

![this](https://cloud.githubusercontent.com/assets/8850830/19417650/2689a11a-93aa-11e6-82df-cab4600ec6d8.png)

**Comment by darkred** [comment]
Thank you
